### PR TITLE
[17.0][MIG] website_sale_b2x_alt_price: Migration to 17.0

### PR DESCRIPTION
**Pull request by carlos-lopez-tecnativa**
@Tecnativa TT50326
cc: @pedrobaeza @CarlosRoca13 @pilarvargas-tecnativa

### CODE REVIEW

**Comment by carlos-lopez-tecnativa** [comment]
@cem-adhoc could you please take a look at the last [commit](https://github.com/OCA/e-commerce/pull/960/commits/968f6190e62839caaf396a7ea19e04f94f8ecbb5) I migrated the tour because it was failing and causing errors in other modules.
Let me know if this commit should be separated from this PR, and if I need to create a new PR just for this fix.
error test

File: website_sale_b2x_alt_price/readme/CONFIGURE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+To choose if you want to display or not the alternative price in the
+website products list, just enable or disable the feature from the
+"Customize" menu. This can be done independently per website:
+
+![toggle-list](../static/description/toggle-list.gif)
```
Comment (pilarvargas-tecnativa) [inline website_sale_b2x_alt_price/readme/CONFIGURE.md:5]: You must update this configuration and its gif. It is now located in the "Customize" tab of the web editor.

File: website_sale_b2x_alt_price/readme/CONFIGURE.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+You can also toggle the behavior similarly in the product details page.
+This can also be done independently per website:
+
+![toggle-detail](../static/description/toggle-detail.gif)
```
Comment (pilarvargas-tecnativa) [inline website_sale_b2x_alt_price/readme/CONFIGURE.md:10]: Same as above

File: website_sale_b2x_alt_price/readme/CONFIGURE.md (hunk 1 above)
Comment (carlos-lopez-tecnativa) [reply website_sale_b2x_alt_price/readme/CONFIGURE.md:5]: @pilarvargas-tecnativa I updated the Configure and added the new GIF. Please let me know your comments.

**Comment by carlos-lopez-tecnativa** [comment]
@yajo you might be interested in taking a look at this PR. Let me know what you think.

**Review by pilarvargas-tecnativa — APPROVED** [review]

**Comment by pilarvargas-tecnativa** [comment]
thanks!

**Comment by pedrobaeza** [comment]
Please squash both migration commits into one.

**Comment by carlos-lopez-tecnativa** [comment]
> Please squash both migration commits into one.

The last commit is related to another module(to pass test), `website_sale_suggest_create_account`. Should it be squashed into the first commit?

**Review by pedrobaeza — APPROVED** [review]
Sorry, I saw it bad.

/ocabot merge nobump

**Pull request merged**